src/config/config.js

```diff
@@ -191,11 +185,11 @@ const config = convict({
     default: 'arn:aws:sns:eu-west-2:000000000000:secret_management',
     env: 'SNS_SECRETS_MANAGEMENT_TOPIC_ARN'
   },
-  snsRunWebShellTopicArn: {
-    doc: 'SNS Run WebShell Topic ARN',
+  snsRunTerminalTopicArn: {
+    doc: 'SNS Run Terminal Topic ARN',
     format: String,
-    default: 'arn:aws:sns:eu-west-2:000000000000:run-webshell-topic',
-    env: 'SNS_RUN_WEBSHELL_TOPIC_ARN'
+    default: 'arn:aws:sns:eu-west-2:000000000000:run-terminal-topic',
+    env: 'SNS_RUN_TERMINAL_TOPIC_ARN'
```

Comment:
If we rename this we'll need to make sure we rename the cdp-app-config setting for it.
The queue it actually uses will still be called run-webshell-topic in the real envs unless we recreate them

Reply:
ah, good spot. I will revert?